exasol_advanced_analytics_framework/temporary_operations/temporary_bucketfs_file_manager.py

```diff
@@ -1,19 +1,19 @@
+from typing import Any
+from pathlib import PurePosixPath
+from exasol_bucketfs_utils_python.bucketfs_location import BucketFSLocation
 
 
 class TemporaryBucketFSFileManager:
-    def __init__(self):
-        pass
+    def __init__(self, bucketfs_location: BucketFSLocation, bucketfs_path: str):
+        self.bucketfs_location = bucketfs_location
+        self.bucketfs_path = bucketfs_path
 
-    def store_states(self):
-        # TODO: states? user event handler class & temporary tables names & ..
-        pass
+    def save_object(self, object_to_load: Any, file_name: str) -> None:
+        self.bucketfs_location.upload_object_to_bucketfs_via_joblib(
+            object_to_load, str(PurePosixPath(self.bucketfs_path, file_name)))
 
-    def load_states(self):
-        pass
+    def load_object(self, file_name: str) -> Any:
+        self.bucketfs_location.download_object_from_bucketfs_via_joblib(
+            str(PurePosixPath(self.bucketfs_path, file_name)))
 
```

Comment:
Besides the functions we already have, we need the following `register_temporary_file` and `register_temporary_directory`  (this one doesn't create the file, but it registers one and return the location). Further, we probably need `save_fileobj` and `load_fileobj`.

Reply:
what does registering to a file ('register_temporary_file' and 'register_temporary_directory') meaning?